pkg/controller/certificaterequests/selfsigned/selfsigned.go

```diff
@@ -150,11 +148,7 @@ func (s *SelfSigned) Sign(ctx context.Context, cr *cmapi.CertificateRequest, iss
 	}
 
 	var template *x509.Certificate
-	if !utilfeature.DefaultMutableFeatureGate.Enabled(feature.DisallowInsecureCSRUsageDefinition) {
-		template, err = pki.DeprecatedCertificateTemplateFromCertificateRequestAndAllowInsecureCSRUsageDefinition(cr)
```

Comment:
You can also remove this function now: DeprecatedCertificateTemplateFromCertificateRequestAndAllowInsecureCSRUsageDefinition

Reply:
ok check again

Reply:
> You can also remove this function now: DeprecatedCertificateTemplateFromCertificateRequestAndAllowInsecureCSRUsageDefinition

ok I will check again

Reply:
> > You can also remove this function now: DeprecatedCertificateTemplateFromCertificateRequestAndAllowInsecureCSRUsageDefinition
> 
> ok I will check again


I checked it one more time, but I don't understand. Afterwards, the template structure and the variables within this structure will continue to be referenced. Should I erase it?

Reply:
> You can also remove this function now: DeprecatedCertificateTemplateFromCertificateRequestAndAllowInsecureCSRUsageDefinition

ah! Let me correct you. The DeprecatedCertificateTemplateFromCertificateRequestAndAllowInsecureCSRUsageDefinition() function has already been deleted and replaced by the CertificateTemplateFromCertificateRequest() function, leaving only this function. Could you please check again?

Reply:
The function is indeed no longer used, but the function definition is still there: https://github.com/yj-yoo/cert-manager/blob/remove_deprecated_function/pkg/util/pki/certificatetemplate.go#L356

Reply:
> The function is indeed no longer used, but the function definition is still there: https://github.com/yj-yoo/cert-manager/blob/remove_deprecated_function/pkg/util/pki/certificatetemplate.go#L356

that's right! I wrote this PR to solve that problem, and I've already deleted that function. Could you check again?